### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by itaysaadia**
This is The more detailed version of the README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,55 @@
...
+
+First, you sould download the source code from github.
+```
+git clone https://github.com/FMFapp/fmf-backend.git 
+```
+
+To test the server run this commands:
+```
+cd fmf-backend/fmf_backend/
+python manage.py runserver
```
Comment (MeitarR) [inline README.md]: also, need to apply the migrations before starting the server

File: README.md [hunk 2]
```diff
@@ -1,2 +1,55 @@
...
+FMF-App is a mobile and web application which was targeted to help to pations, doctors and researchers of the Familial Mediterranean fever.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+To run this server on your machine you need to download the requirements from the requirements.txt file. 
+is is recommended to use venv insead of your regular interpreter and use the latest version of Python 3.
```
Comment (MeitarR) [inline README.md]: "is is" -> It is 
"insead" -> instead

File: README.md [hunk 3]
```diff
@@ -1,2 +1,55 @@
-# fmf-backend
-RESTful API server
+# FMF-App Backend
+
+This is the back-end side of the FMF-App application. 
+FMF-App is a mobile and web application which was targeted to help to pations, doctors and researchers of the Familial Mediterranean fever.
```
Comment (MeitarR) [inline README.md]: "to help to' -> to help
"pations" -> patients
"doctors and" -> doctors, and

File: README.md [hunk 4]
```diff
@@ -1,2 +1,55 @@
...
+is is recommended to use venv insead of your regular interpreter and use the latest version of Python 3.
+```
+python -m pip install -r  requirements.txt
+```
+
+### Installing
+
+A step by step series of examples that tell you how to get a development env running
+
+First, you sould download the source code from github.
```
Comment (MeitarR) [inline README.md]: "sould" -> should

File: README.md (hunk 1 above)
Comment (itaysaadia) [reply README.md]: Good point, I will add is to the code

File: README.md [hunk 5]
```diff
@@ -1,2 +1,57 @@
...
+
+First, you should download the source code from github.
+```
+git clone https://github.com/FMFapp/fmf-backend.git 
+```
+
+To test the server run this commands:
+```
+cd fmf-backend/fmf_backend/
+python manage.py makemigrations
```
Comment (MeitarR) [inline README.md]: Remove this line. 
When they clone the repo they already have the migrations that we generated and they didn't change anything that requires generate new migrations

**Pull request merged**